projects/storefrontlib/src/cms-components/cart/cart-coupon/cart-coupon-anchor/cart-coupon-anchor.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit } from '@angular/core';
+import { CartCouponAnchorService } from './cart-coupon-anchor.service';
+
+@Component({
+  selector: 'cx-cart-coupon-anchor',
+  templateUrl: './cart-coupon-anchor.component.html',
```

Comment:
Cna we set `changeDetection: ChangeDetectionStrategy.OnPush,` here for performanc?

Reply:
Done